bob-common/src/metrics/mod.rs

```diff
@@ -168,6 +175,21 @@ impl BobClient {
             timer.elapsed().as_nanos() as f64
         );
     }
+
+    pub(crate) fn delete_count(&self) {
+        counter!(self.prefix.clone() + ".delete_count", 1);
```

Comment:
This string concatenation executes for every request, but it is obviously that every time it gives the same result. I suggest to precalculate all strings in constructor. Probably, there should be a separate struct with counter names, stored inside `BobClient` and wrapped with `Box`

Reply:
Although I've removed string concat, we still need to clone string every time (as metrics requires owned strings).